src/ensembl/src/content/app/entity-viewer/EntityViewer.tsx

```diff
@@ -91,11 +86,13 @@ const EntityViewer = (props: Props) => {
         genomeId: activeGenomeId,
         entityId: entityIdForUrl
       });
-      props.replace(replacementUrl);
+      dispatch(replace(replacementUrl));
```

Comment:
I noticed that GeneView.tsx also dispatches the same way as how you've done here. So if we change the code here then we'll need to change it there as well: https://github.com/Ensembl/ensembl-client/blob/7394a9ca5bd18472eab2bf30d23147021e2bd306/src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.tsx#L261

Reply:
> However, this component is not one anymore
I am not sure I understand...

I don't know whether connected react router turned out to be of much use. I remember we wanted to have access to the current url in redux; but I don't think we ever used this functionality since. The only place I can think of that we are definitely still using is the analytics middleware that reports all url changes to our analytics.

We are currently consistently using connected react router in all functions that change the route programmatically. We can audit our app to see whether we need connected-react-router; and remove it if we don't; but until then I suggest we keep using it purely for consistency's sake.

Reply:
@azangru I thought that we use it along with connected components to access the router functionality.  So I've misunderstood why we are still using connected-react-route.

@jyothishnt Let's leave it as it is.